Text-Based-Adventure/Text-Based-Adventure/main.swift

```diff
@@ -0,0 +1,459 @@
+/**/*/**/*/******,      .*##%%%%&&&%/,*//*,,//(//,,/*/%&&&&%(***,,,,,,,,,,,*/%%#(((/*////**//*,.,*,,*/#%%//(////*,,******,,,,..............
+/*//////(////(///*      .*%%%%%%%%%##((##/.....*(((###&&&%(****,,,,,,,,,,,,,,,*#%%%#(((((*.....,*////////(//////*********,*,,,,,,.,,.......
+////////(////(**/*        .,(%%%#//(((###%%%%%###%%%&&%(/**((/**,,,,,,,,,,,**/(/*/###/////(((((((//*****/###**********,,.....,,,...,.......
+////////////////**           ,#%(#%%###%%%%%%%%%%#####%%#**///(((/////////((///(#(////*/((///////***//(((/(#**,,,**,,,.....,,..............
+/*//////(//(/(///*          ..,(%%%%%%%%%%%%%%(((/*(#(#%%##*((/(/////*///(////**///**/*,*///(#####(//(/((//*,,,**,,,**,*/((//,,............
+/**/*/,**********,         .*,*#%%%%%%&%#/(/,,/((*,.**(%%%%%####((((((((((##((#(((/****,,,**,,,//#%#/////((/***,*,,,,,,,,,,/(*,,...........
+/****************,         .###%%%%%%&%(/*.. ..*,**.*,.*(%&%%%#################(///***,,,,,,,,,,,,(%%#///*////,,,,,,,,,,.,,**/,,...........
+/((////*(////(///*         ,#%##%%%%&%###///**///((**/#&&&&&%%###(((((((((#%%%#####***/***,,*/**/(//(#(///////,,,..,,.,,.,,,,*/*,..........
+//////*/////////** .... .   .*/###%%###((((//////(#%&%%&&&%%(/////********///(#%%###%%#///**///((///**//*****,,.,.....,,.,,,.,//,,.........
+/((**/**(//(/(*//*.*////////****(%%%########%%%%%###((%&%#(*,*(/***////***/(/**/**((///(((((((((((/////##*,,,,,..,,..,,,..,,,.,/*,,........
+///***,**************************/(#%%%%%%%%%%%%%%%###(((((#(/******///***/**,,**//////(/////****///(((//*,****,,,,,,,,,,,,,,..,/*,,.,,,,,,
+/*/*****/*********************/%%#%%%%%%%%&%**,**/(/,*(##%%#//////////**((/**/(((/**///******(%#(////((//##/,,,*,,,*,,,,,,,,*,.,/#*,,,,,..,
+/(((//**(//(/(/////***********(#####%%%%%&%(,..*,,*/,,/,,(#%%%%%%(((((((####(((/**,,,*,,,,,,,*(%%(//*/(((((*,,,,*,,*,,,,,,,,,*,,*(#,,..,,,,
+///////////////////***********/#%##%%%%%%%##//,**,*(*,/**(#&%%%##(((((((((#####(//*,,**,****/((%%#//**/((/*,,,,,,,,,,,,,,,,,,,,,,*#/,,,,..,
+////(/**(////(**//****************#######%#((#((((((/(#&&&&&&%##//////////(%%%%%##(******/((*,,***//*//****,,,,,,,,,,**,,,,.,,,,,,(#/,,,..,
+//////**///////*****************,,/*/%%/(##########(//#%%%(/**,,****,,**,*/###(/**////////******/#(***,,,,**,,,,,,,***.,,..,,,,,*%#,.,,,,
+/*/**/*//*/***********************####(#%%%%%%%%%%%%%%#(/(/**///*///*//(((*,,.,*((//////((/////(((**(((**,,,***,,*,,.,**,.,,,..,**,(%#*.,,,
+(///////(//(/(////****************##%(#%%%%%##%%%(////#((((/*/(((((/////((*,*//******///(#(////((((//(#**,,,***,,**,,,***..*,,,,,*,*(%#,,.,
+####((/((//////////*/***//*/****/*(##%%%%%%%%%&%(,,,,,/**##%%%//**********(((((//*,,**,,*/%%////((((//(,**,,,,**,,*,,,***,,,*,,,,,,,*%%/.,*
+""")
+    print("Okay \(enterName ?? "playa"), a \(howAdenturousAsAnInt) is adventurous enough to try the Vessel at Hudson Yards!!!......")
+    print("Ready to start? \"yes\" or \"no\"?")
```

Comment:
If I type in "no" it ends the game, it should have an option to either ask again or redirect to another path.